### PR TITLE
fix: fix Lighthouse errors (PWA config)

### PR DESCRIPTION
**Pull request by ArturLange**
### What
- Added standard PWA compatible icons that were missing.
- Fixed some other errors from Google Lighthouse (accessibility etc.)
- Changed open graph image to absolute path as FB does not recognise relative image - see below
Also see #32 

image

### CODE REVIEW

File: src/components/Header.vue [hunk 1]
```diff
@@ -2,21 +2,21 @@
...
         <template v-slot:activator="{ props }">
           <v-btn v-bind="props" icon="mdi-account-circle"></v-btn>
         </template>
         <v-list>
           <v-list-item :slim="true" prepend-icon="mdi-account" disabled>{{ username }}</v-list-item>
           <v-divider></v-divider>
-          <v-list-item :slim="true" prepend-icon="mdi-view-dashboard-outline" to="/dashboard">{{ $t('Header.Dashboard') }}</v-list-item>
-          <v-list-item :slim="true" prepend-icon="mdi-cog-outline" to="/settings">{{ $t('Header.Settings') }}</v-list-item>
-          <v-list-item :slim="true" prepend-icon="mdi-logout" @click="signOut">{{ $t('Header.Sign-out') }}</v-list-item>
+          <v-list-item aria-label="Dashboard" :slim="true" prepend-icon="mdi-view-dashboard-outline" to="/dashboard">{{ $t('Header.Dashboard') }}</v-list-item>
```
Comment (raphodn) [inline src/components/Header.vue:17]: we should use the translations here

File: .gitattributes
Comment (raphodn) [inline .gitattributes:1]: can you explain the purpose ?

**Review by raphodn — APPROVED** [review]
thanks !

File: .gitattributes
Comment (ArturLange) [reply .gitattributes:1]: I added image files to Git LFS.
AFAIK adding them to Git repository normally can bloat the repository over time, this should be safer

Comment (raphodn) [reply .gitattributes:1]: it seems to be an [addon](https://stackoverflow.com/questions/71745178/gitlfs-images-and-or-files-how-do-i-view-them-how-do-i-download-them), and only for git, and only for a couple of images here. I don't think we need this ;)

Comment (ArturLange) [reply .gitattributes:1]: You're right, it might not be worth it :)

**Review by teolemon — APPROVED** [review]
Swell

File: public/android-chrome-192x192.png
Comment (raphodn) [inline public/android-chrome-192x192.png:1]: I have an error in my console when running this branch locally

image

Comment (raphodn) [reply public/android-chrome-192x192.png:1]: and github says 

image

Comment (ArturLange) [reply public/android-chrome-192x192.png:1]: Good catch, they were messed up by adding and removing LFS :crying_cat_face: 
I added them again, they should work now.

**Pull request merged**